src/python/ensembl/io/genomio/__init__.py

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 """Genome Input/Output (GenomIO) handling library."""
 
-__version__ = "1.4.0"
+__version__ = "1.4.1"
```

Comment:
```suggestion
__version__ = "1.5.0"
```
You are adding new functionality, so this is a more adequate version increment for that.